Sources/GRPCCodeGen/CodeGenerationRequest.swift

```diff
@@ -109,7 +125,7 @@ public struct CodeGenerationRequest {
       /// Represents the imported item's kind.
       public struct Kind {
         /// Describes the keyword associated with the imported item.
-        internal enum Value {
+        internal enum Value: CustomStringConvertible {
```

Comment:
A bit of a nit, but `ImportDescription.Kind`, which basically mirrors this type, has a `String` raw value instead of conforming to `CustomStringConvertible`. Is there a reason why this enum conforms to `CustomStringConvertible` instead of doing the same as `ImportDescription.Kind`?

Reply:
Not really, I will use raw values instead in both cases